tests/php/ORM/DBSchemaManagerTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace SilverStripe\ORM\Tests;
+
+use ReflectionMethod;
+use SilverStripe\Core\Config\Config;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\ORM\Connect\DBSchemaManager;
+use SilverStripe\ORM\Tests\DBSchemaManagerTest\ChildClass;
+
+class DBSchemaManagerTest extends SapphireTest
+{
+    protected $usesDatabase = false;
+
+    public function provideCanCheckAndRepairTable()
+    {
+        return [
+            // not ignored, but globally not allowed
+            [
+                'tableName' => 'DBSchemaManagerTest_SomeModel',
+                'globalAllowCheck' => false,
```

Comment:
```suggestion
                'checkAndRepairOnBuild' => false,
```
do for all the others too

Reply:
Done